### PR TITLE
Added a new story on experience

### PR DESCRIPTION
**Pull request by SupriyaSirbi**
Added new story.

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1838)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/experience/lessons-learnt/lessons-learnt.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+You are walking on a street,you stumble across a rock and fall down.
```
Comment (casey-collab) [inline english/experience/lessons-learnt/lessons-learnt.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**